scripts/mod_loader/altered/misc.lua

```diff
@@ -1,6 +1,48 @@
 
+-- modified vanilla save_table with optional 'depth' parameter
+function save_table(target, depth)
+	local ret = "{"
+	for i, v in pairs(target) do
+		local value = ""
+	    if type(v) == "table" then
+	        if v ~= target.__index then
+				if not depth or depth > 0 then
+					depth = depth and depth - 1 or nil
+					value = save_table(v, depth)
+				else
+					value = "table"
+				end
+			end
+		elseif type(v) == "userdata" and v["GetLuaString"] ~= nil then
+			value = v:GetLuaString()
+        elseif type(v) ~= "userdata" and type(v) ~= "function" then
```

Comment:
Seems to have some mixed tabs and spaces here. Github has that annoying "feature" where tabs show as 6 spaces, so it stands out. I think the rest of the file does tabs.

Reply:
Right you are! Maybe this "feature" is there to make it apparent.
It also revealed a bug when testing the function again, so that's an extra bonus